src/osd/OSD.h

```diff
@@ -1620,6 +1624,11 @@ class OSD : public Dispatcher,
 	p->complete(0);
       }
     }
+
+    double get_cost_per_io() const {
```

Comment:
Same deal here, move the addition of this helper and supporting changes to a separate commit.